exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,9 @@
-// Write your function here
+function createGreeting(firstName) {
+    var message = "Hello, my name is " + firstName;
+    return message;
```

Comment:
For functions that are small and only do one operation (such as this one that does a string concatenation), you don't need to assign it to a variable first. You can just return the statement from the function. It's a bit cleaner and easier to read.

E.g.
```
function createGreeting(firstName) {
    return "Hello, my name is " + firstName;
}
```